README.md

```diff
@@ -105,10 +105,16 @@ quality checks and releases.
    being merged into the main branch.
 
 2. **Release Workflow**
-   When changes are pushed to the `main` branch, this workflow handles versioning and releases using
+   When changes are pushed to `main`, this workflow handles versioning and releases with
    `semantic-release`. It automatically bumps the version, generates release notes, creates a tag,
-   and publishes a Docker image to an Amazon ECR repository.
-
+   and publishes a Docker image to Amazon ECR. This new version is automatically deployed to the
+   development environment.
+
+3. **Production deployment**
```

Comment:
I find it weird to have a third item here, when line 99 mentions only two workflows. Maybe this should just be a paragraph?

```suggestion
```
And remove the indentation on the paragraph below

Reply:
I changed the sentence instead.